fobi_custom/override_theme/templates/override_theme/edit_form_entry_ajax.html

```diff
@@ -0,0 +1,98 @@
+{% load i18n fobi_tags %}
+
+<div class="col col-lg-11 card">
+  <div class="card-body">
+
+   <h2 class="card-title">{% trans "Edit" %} <span class="font-italic">{{form_entry.surveyformentry.name}}</span></h2>
+   <small class="form-text text-muted">Add, edit, delete, and reorder survey questions</small>
+   <br />
+
+   <ul class="nav nav-tabs" id="surveyTabs" role="tablist">
+     <li class="nav-item">
+       <a class="nav-link active" id="surveyQuestionsTab" data-toggle="tab" href="#survey-questions" role="tab" aria-controls="survey-questions" aria-selected="true">Questions</a>
+     </li>
+     <li class="nav-item">
+       <a class="nav-link" id="surveyPropertiesTab" data-toggle="tab" href="#survey-properties" role="tab" aria-controls="survey-properties" aria-selected="false">Properties</a>
+     </li>
+   </ul>
+
+   <div class="tab-content" id="surveyTabsContent">
+
+     <div class="tab-pane fade show active" id="survey-questions" role="tabpanel" aria-labelledby="survey-questions">
+       <div class="panel-body">
+         <br />
+
+         <div class="form-group row action-box">
+          <label for="formElementDropdown" class="col-form-label col-lg-4">
+              <i class="fas fa-plus-circle icon-pink"></i>&nbsp;
+              Add a new survey question:
+          </label>
+
+          <div class="col-lg-8">
+            <select id="formElementDropdown" class="select form-control">
+              <option>--------</option>
+              {% for group, plugins in user_form_element_plugins.items %}
+              <optgroup label="{{group}}">
+                {% for form_element_uid, form_element_name in plugins %}
+                <option value="{% url 'fobi.add_form_element_entry' form_entry.pk form_element_uid %}">{{ form_element_name }}</option>
+                {% endfor %}
+              </optgroup>
+              {% endfor %}
+            </select>
+          </div>
+          </div>
```

Comment:
Could we display this field below the survey questions? That way it'll be clear that you're adding new questions to the bottom of the survey, not the top.

Reply:
sure!

Reply:
I moved the "Add a new survey question" field to the bottom of the form and it didn't sit right with me, either above or below the "Save ordering" button. Above it felt too much like part of the survey, and below it felt like an afterthought instead of one of the main actions of the page. If it's alright with you I'd like to try user testing this with "Add a new survey question" at the top of the page.

Reply:
Ok, let's punt this to user testing!